src/Mod/CAM/Roadmap/Functionality/Operation Configuration.md

```diff
@@ -16,7 +16,7 @@
 | ------------------- | ---------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Entry Strategies    | Provide helical and ramp entry           | Partially complete via dressup. <br>specific strategies are possible but limited and buggy                                                                                                                                                                                                                                                                                                                                                           |
 | Entry Start Point   | Allow user to specify entry point        | Start point selection is poorly implemented.  It's not allowed at all in adaptive.<br>When used in other pocket ops (millface) only a single point is allowed.  If the pocket algorithm needs to retract and re-enter, it will use whichever point is needed.  <br><br>Pocket entry point should be studied to evaluate the current state-of-the-art.  Either remove it and let the op set start point automatically or reimplement it consistently. |
-| Clearing Strategies | zigzag, offset, adaptive                 | COMPLETE.                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| Clearing Strategies | zigzag, offset, adaptive                 | COMPLETE.  Fix arealib and restort Spiral clearing.                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
This section should be agnostic about implementation.  It's talking about what the functionality should be in any CAM application.  Not a list of todo items.  

The requirement for a spiral clearing strategy is valid.

Can you remove the area lib stuff and expand on the spiral clearing requirement.

Reply:
OK , I know you'd rather replace arealib with python.  Since no one can read arealib, there is a case for that. 
Is it withing parametes to say that Spiral used to exist but bugs in arealib caused it to be removed. 
IIRC there was  a second method which got removed at the same time : triangle?  I never used it.

Reply:
Try to separate what needs to be done.  "We need a spiral clearing strategy for pockets"  from HOW it gets done.
Arealib?  That's one possibility.  CavContours?  Custom code using OCC core?  Something I haven't heard of yet?  Who knows.  

Smart people can debate and select the correct implementation later.

Reply:
I've made a new PR with "fix Spiral" instead of fix arealib.